assets/src/blocks/Accordion/AccordionBlock.js

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable no-unused-vars */
+import { AccordionEditor } from './AccordionEditor';
+import { frontendRendered } from '../frontendRendered';
+import { CSS_VARIABLES_ATTRIBUTE } from '../CssVariablesAttribute';
```

Comment:
This import is not used, can be removed 🙂

Reply:
right.. I am removing it, I used it in the beginning for the multiple style options of the block :)